<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatting</class>
 <widget class="QDialog" name="chatting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="send_pushButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>393</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userName_lineEdit">
   <property name="geometry">
    <rect>
     <x>204</x>
     <y>0</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pictureSelect_pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>393</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>图片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chatRecord_pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>393</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>聊天记录</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="sendReady_textEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>420</y>
     <width>521</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="connect_pushButton">
   <property name="geometry">
    <rect>
     <x>467</x>
     <y>0</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="showContent_textEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>33</y>
     <width>521</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
